pallets/did/src/lib.rs

```diff
@@ -126,12 +125,9 @@ pub mod pallet {
 		/// A DID has been deleted.
 		/// \[transaction signer, DID identifier\]
 		DidDeleted(AccountIdentifierOf<T>, DidIdentifierOf<T>),
-		/// A DID-authorized call has been successfully executed.
-		/// \[DID caller]
-		DidCallSuccess(DidIdentifierOf<T>),
-		/// A DID-authorized call has failed to execute.
-		/// \[DID caller, error]
-		DidCallFailure(DidIdentifierOf<T>, DispatchError),
+		/// A DID-authorized call has been executed.
+		/// \[DID caller, dispatch result]
```

Comment:
```suggestion
		/// \[DID caller, dispatch result\]
```

Reply:
Fixed in https://github.com/KILTprotocol/mashnet-node/pull/207/commits/2155d5e2496391fa181646e89343cee9d4f1c69e